### PR TITLE
Show process template errors as toasts

### PR DESCRIPTION
**Pull request by spadgett**
https://trello.com/c/6WVJdgjL

### CODE REVIEW

File: app/scripts/directives/processTemplate.js [hunk 1]
```diff
@@ -152,16 +153,32 @@ function ProcessTemplate($filter,
     modalInstance.result.then(createResources);
   };
 
+  var alerts = {};
+  var hideNotificationErrors = function() {
+    NotificationsService.hideNotification("process-template-error");
+    _.each(alerts, function(alert) {
+      if (alert.id) {
```
Comment (jwforres) [inline app/scripts/directives/processTemplate.js]: this assumes that we will only ever put id's on the error alerts, should this also check that the alert is a warning or error alert

**Comment by spadgett** [comment]
@jwforres Thanks, updated.

File: app/scripts/directives/processTemplateDialog.js [hunk 2]
```diff
@@ -80,6 +79,8 @@
     };
 
     ctrl.close = function() {
+      $scope.$broadcast('hideTemplateNotificationErrors');
```
Comment (jwforres) [inline app/scripts/directives/processTemplateDialog.js]: assuming i'll need to do the same thing from my overlay PR then

**Comment by jwforres** [comment]
[merge]

File: app/scripts/directives/processTemplateDialog.js (hunk 2 above)
Comment (spadgett) [inline app/scripts/directives/processTemplateDialog.js]: Yeah, and something similar is needed for `from-file` when #1677 goes in.

**Pull request merged**